### DISCUSSION TITLE
Is there a (defined) period with which new Typst versions are released?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [typst/typst#2798] [Q&A]
I have noticed that (last) versions are released with a period of about 5–6 (7) weeks, so about 1.25–1.5 month. But I want to know if there is anything like this really exist. How next release date is usually chosen?

**Comment by laurmaedje** [comment]
There is no fixed cycle so far. We have decided on a case by case basis when a good point for a release is. 0.10 will come very soon and I intend to let some time pass until 0.11 (maybe 2 months) because releases always take a lot of time and I want to focus on getting stuff done in the compiler. That's why I am putting more work into making 0.10 as stable and solid as it can be.

**Reply by Andrew15-5** [reply]
> 0.10 will come very soon

Yes, according to "the statistics" I expect it to be available in the next week.

> There is no fixed cycle so far

I assume that after a v1.0.0 the Typst compiler will go into maintain mode or new releases will be much less often (I don't know if v2 with some big new features will be a thing). So we can talk about beta development specifically. And you have said that for _some things_ current "cycles" are too fast. And with this in mind, are there any plans on creating a specific cycle length (like with Java and Python)? Maybe every month? 2? 3? Or does "unpredictable beta development stage" prevents from making any concrete periods?
